index.html

```diff
@@ -84,6 +85,15 @@
             <img class="hero__img" src="img/hero2.jpg"/>
             <h1 class="hero__title hero__title--withShadow">About Us</h1>
         </section>
+
+        <section class="cta">
+            <h2 class="cta__header">Proudly Serving Couples for 30 Years</h2>
+            <div class="cta__info">
+                <p class="cta__info-text">The Wedding Shop is a multi-award-winning gift list service that has been supporting couples for three decades. It’s often a whirlwind of wedmin and emotions, but we’re here to help, and thrilled to be a part of your exciting journey.</p>
+                <button class="cta__info-btn">Our Service Explained</button>

```

Comment:
the whole naming convention here is fine but I'd change the section class name to something that would suggest that it's, well, a section. 'CTA' indicates that it's a CTA so a button/link.

Reply:
also, the CTA - meaning the button - is a component that for sure will be used across the whole site, it shouldn't be just an element of this section but a block itself (it can be both at the same time). So e.g. 
`<button class="button section__button"> ...` - the `button` classname would include all the generic styles, while `textSection__button` would include only the ones that apply to this particular section (if they're needed at all).
This way you can use .button for buttons in any place you want. I hope it makes sense.

Reply:
Ok, have changed cta to service - do you think that is ok?